src/sessions/video-session-handler.ts

```diff
@@ -126,6 +126,16 @@ export default class VideoSessionHandler extends BaseSessionHandler {
       .filter((receiver) => receiver.track && receiver.track.kind === 'video')
       .map((receiver) => receiver.track.id);
 
+    /**
+     * Firefox messes the trackIds up from what is actually in the sdp offer.
+     * Need to pull it from the offer to accurately match the track.sinks
+     */
+    const incomingVideoMsidTrackId = this.getTrackIdFromSdp(session.pc.pc.remoteDescription.sdp, 'video');
+
+    if (incomingVideoMsidTrackId) {
+      incomingVideoTrackIds.push(incomingVideoMsidTrackId);
```

Comment:
Pushing this onto the array will cause duplicate trackIds in chrome. I thought about adding logic to not add duplicates, but I figured the `intersection(...)` call later would remove those anyway. Open for opinion. (This happens in the function below too).